tests/acceptance/stepDefinitions/publicLinkContext.js

```diff
@@ -182,3 +182,23 @@ When('the user closes the public link details sidebar', function () {
     .filesList()
     .closeSidebar(100)
 })
+
+When('the user copies the url of public link named {string} of file/folder/resource {string} using the webUI', async function (linkName, resource) {
+  await client.page.FilesPageElement
+    .filesList()
+    .closeSidebar(100)
+    .openPublicLinkDialog(resource)
+  return client.page.FilesPageElement.publicLinksDialog()
+    .clickCopyLinkURI(linkName)
+})
+
+When('the user navigates to the copied public link using the webUI', function () {
```

Comment:
this is the same as `When('the user navigates to the copied private link using the webUI', function () {` please DRY
```suggestion
When('the user navigates to the link in clip-board using the webUI', function () {
```

Reply:
:rocket: